style.css

```diff
@@ -1,8 +1,92 @@
 body {
-  box-sizing: border-box;
   margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.nav__bar {
+  background: #64b5f6;
+  color: white;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 2rem 0 2rem;
+}
+
+.auth__name {
+  text-align: left;
+  margin-left: 2rem;
+}
+
+.links {
+  margin-right: 3rem;
+
+  text-decoration: none;
+  color: white;
+  /*float: right;*/
+  align-items: center;
+
+
+}
 
+.nav-right {
+  text-align: right;
 }
-*, *:before, *:after {
-  box-sizing: inherit;
+
+.container {
+  /*grow up to 65rem*/
+  max-width: 70rem;
+  /*center if there is width left*/
+  /*margin: auto;*/
+  margin: 0 4rem  0  8rem;
+}
+
+.card {
+  /*width: calc(100% / 2 - 1rem - 10rem);*/
+  box-shadow: .2rem .2rem .4rem;
+  float: left;
+  max-width: 25rem;
+  position: relative;
+  margin: 3rem;
+}
+
+.post__title {
+  text-align: center;
+  background: #ffd740;
+  padding-top: 5rem;
+  padding-bottom: 5rem:
+}
+
+.post {
+  text-align: center;
+}
+
+.img {
+  max-width: 100%;
+  border-radius: 50%;
+  position: absolute;
+  z-index: auto;
+  /*top: 14.5%;
+  left: 3%;*/
+  position: absolute;
+  margin-left: -3rem;
+}
+
+
+.week {
+  position: relative;
```

Comment:
This should be position absolute so it does not take up room compared to other elements.
This is why you are getting the whitespace above the title.